Sources/SpeziOpenAI/MessageInputView.swift

```diff
@@ -101,7 +102,7 @@ struct MessageInputView_Previews: PreviewProvider {
             Color(.secondarySystemBackground)
                 .ignoresSafeArea()
             VStack {
-                MessagesView($chat)
+                MessagesView($chat, hideMessagesWithRoles: MessageView.Defaults.hideMessagesWithRoles)
```

Comment:
if we use the default value in the initializer, we can remove the argument here.